lib/src/google_fonts_base.dart

```diff
@@ -234,7 +235,7 @@ Future<ByteData> _httpFetchFontAndSaveToDevice(
       );
     }
     if (!isWeb) {
-      _saveFontToDeviceFileSystem(fontName, response.bodyBytes);
+      unawaited(_saveFontToDeviceFileSystem(fontName, response.bodyBytes));
```

Comment:
Never seen this before, but isn't _saveFontToDeviceFileSystem this actually an async call?

Reply:
Yeah, this function is just to consciously acknowledge that the future is _not_ `await`ed. Pedantic enforces that all futures are either `await`ed, or marked as intentionally unawaited.